lib/src/main/java/technology/dice/dicewhere/provider/maxmind/decorator/MaxmindVpnDecoratorDbReader.java

```diff
@@ -33,7 +33,8 @@
 
   public MaxmindVpnDecoratorDbReader(
```

Comment:
DecoratorDbReader and it's descendants are not thread safe, due to non-volatile, non-atomic `lastFetched`

Reply:
Please open an issue to address this